tests/robustness/model/describe.go

```diff
@@ -223,6 +223,8 @@ func describeRangeRequest(opts RangeOptions, revision int64) string {
 		return fmt.Sprintf("get(%q%s)", opts.Start, kwargsString)
 	case opts.End == clientv3.GetPrefixRangeEnd(opts.Start):
 		return fmt.Sprintf("list(%q%s)", opts.Start, kwargsString)
+	case strings.HasSuffix(opts.Start, "\x00"):
```

Comment:
Could we make the validation here more precise. Like check that opts.Start is prefix of opts.End without "0" at the end.

Reply:
I check if `opts.End` without trailing `0` is a prefix of `opts.Start`